addon/components/flash-message.js

```diff
@@ -58,6 +58,14 @@ export default Component.extend({
       set(this, 'active', true);
     });
     set(this, 'pendingSet', pendingSet);
+    this.element.addEventListener('mouseenter', this._mouseEnter);
```

Comment:
are you sure that this actually works? it looks like `_mouseEnter()` is called without a `this` reference, so `const flash = get(this, 'flash');` would always return `undefined` 🤔

Reply:
Good catch. I think you are right about this - looks like I forgot to add the action helper. However, I noticed doing so breaks the test (I think because they run on ember 3.8.3). So I used bind() to bind this to the functions to handle the events. Opened #315 to address this